ts/components/session/LeftPaneMessageSection.tsx

```diff
@@ -201,9 +213,78 @@ export class LeftPaneMessageSection extends React.Component<Props, State> {
     );
   }
 
-  private renderClosableOverlay(overlay: SessionComposeToType) {
+  private handleMessageRequestsClick() {
+    this.handleToggleOverlay(SessionClosableOverlayType.MessageRequests);
+  }
+
+  /**
+   * Blocks all message request conversations and synchronizes across linked devices
+   * @returns void
+   */
+  private async handleBlockAllRequestsClick() {
+    let messageRequestsEnabled = false;
+    if (window?.inboxStore?.getState()) {
+      messageRequestsEnabled =
+        window.inboxStore?.getState().userConfig.messageRequests === true &&
+        window.lokiFeatureFlags?.useMessageRequests === true;
+    }
```

Comment:
this `messageRequestsEnabled` should come from the props, using your selector again

Reply:
search for InnerLeftPaneMessageSection